bluesky/broker_callbacks.py

```diff
@@ -25,9 +20,10 @@ class LiveImage(CallbackBase):
     relevant commit is a951b7.
     """
     def __init__(self, field):
+        from matplotlib.pyplot import figure
```

Comment:
Here is a use case that I try to address:  The user logs in over ssh or su and forgets to forward the DISPLAY.  This is not trivial - especially with su, where one needs to adjust the .Xauthority permissions.  The user wants to export some tiff files and does not need any plotting.  The user would get stuck, because the import of broker_callbacks would fail together with a top-level import of matplotlib.pyplot.

I feel it is better if user can accomplish jobs that can be accomplished without DISPLAY.